Initializes the november newsletter (#222)
readthedocs/ethicalads.io: content/posts/newsletter-november-2021.md

Title: EthicalAds Newsletter - November 2021

> Date: December 8, 2021

Reviewer: ```suggestion
Date: December 10, 2021
```

Title: EthicalAds Newsletter - November 2021
Date: December 8, 2021
description: Updates from November, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community
authors: Ra Cohen

> image: /images/posts/2021-november-newsletter.jpg

Reviewer: There's a typo in the filename here. The image is spelled "newsleter" and should be renamed.

image_credit: <span>Photo by our founder <a href="https://twitter.com/ericholscher?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor">Eric Holscher</a></span>

## New features

These are the major features we released in the last month:

> * Rolled out our new [floating ad](https://ethical-ad-client.readthedocs.io/en/latest/#placement-style) type for all publishers as well as implemented its usage on more projects on [Read the Docs](https://docs.readthedocs.io/en/stable/).

Reviewer: ```suggestion
* Rolled out our new [floating ad](https://ethical-ad-client.readthedocs.io/en/latest/#placement-style) type for all publishers as well as implemented its usage on Read the Docs' [own documentation](https://docs.readthedocs.io/en/stable/). This is an opt-in feature that publishers can choose if they want a floating ad in the corner which is a pretty common setup.
```

> * Added a stripe integration so advertisers can view previous invoices in the client. 

Reviewer: ```suggestion
* Added a stripe integration so advertisers can view previous invoices. 
```

"Client" here might confuse with our ad client.
